addons/mail/static/src/discuss/core/public_web/discuss_command_palette.js

```diff
@@ -112,14 +111,17 @@ commandSetupRegistry.add("@", {
     placeholder: _t("Search a conversation"),
 });
 
+/**
+ * @param {string} name
+ * @param {import("models").Store} store
+ */
 async function makeNewChannel(name, store) {
-    const data = await rpc("/discuss/channel/create_channel", {
-        name: name,
-        group_id: store.internalUserGroupId,
-    });
-    const { Thread } = store.insert(data);
-    const [channel] = Thread;
-    channel.open({ focus: true, bypassCompact: true });
+    const { channel } = await store.fetchStoreData(
+        "/discuss/create_channel",
```

Comment:
Why not use something more flexible like `/discuss/channel/create`? Opening the door to create `/discuss/channel/update`, etc.

Reply:
Because it is not the `create` method of the ORM, but a call to the `_create_channel` specific method.